aptos-move/framework/aptos-token/sources/token.move

```diff
@@ -1483,4 +1488,79 @@ module aptos_token::token {
 
         withdraw_with_capability(cap)
     }
+
+    #[test(creator=@0xcafe, another_creator=@0xde)]
+    fun test_burn_token_from_both_limited_and_unlimited(
```

Comment:
Can you separate this into 2 tests? I don't see why these 2 flows need to be in the same tests.

Reply:
This is to reduce the boilerplate code.  too many tests now.